CONTRIBUTING.md

```diff
@@ -26,15 +26,42 @@ In order to work with the repo offline you neet to clone the code to your local
 
 Create a new branch for your changes on your local Git repository. You should make it a descriptive name, like ‘feature_new-project’, ‘fix_broken-links’ or similar.
 
-### Step 3: Install Jekyll
+### Step 3: Build and serve the website locally
 
-Run `bundle install` in the root directory of your local Git repository directory. This will install all of the requirements for running the site locally. 
+For properly reviewing any changes made to your local copy of the MEI website in your browser, you need Jekyll to build and serve the page on your local machine. There are two recommended options for doing this:
 
-Windows user should check out this step-by-step guide to setting up Jekyll on Windows: [https://jekyll-windows.juthilo.com/](https://jekyll-windows.juthilo.com/)
+a. Run Jekyll on your machine natively
+b. Run Jekyll in a Docker container
 
-### Step 4. Run the Jekyll server
+#### Step 3a: Run Jekyll on your machine natively
 
-You can run the Jekyll service locally with `bundle exec jekyll serve --baseurl=""`. This will compile the site and start serving the content locally. Since compiling can take a while, you can add the option `-V` to get a more verbose output and information about the running compilation steps. Once it's ready, you can open your browser and visit `http://localhost:4000` to see your local version of the site.
+To install and run Jekyll natively on your machine, please follow these steps:
+
+  1. Install Jekyll
+
+  Run `bundle install` in the root directory of your local clone of this repository. This will install all of the requirements for running the site locally.
+
+  Windows users should check out this step-by-step guide to setting up Jekyll on Windows: [https://jekyll-windows.juthilo.com/](https://jekyll-windows.juthilo.com/)
+
+  2. Run the Jekyll server
+
+  You can run the Jekyll service locally with `bundle exec jekyll serve --baseurl=""`. This will compile the site and start serving the content locally. Since compiling can take a while, you can add the option `-V` to get a more verbose output and information about the running compilation steps. Once it's ready, you can open your browser and visit `http://localhost:4000` to see your local version of the website.
+
+#### Step 3b: Run Jekyll in a Docker container
+
+To avoid installing Jekyll natively and to assure it comes with all the necessary dependencies, you can run it in a preconfigured container image. The publishers of Jekyll offer a corresponding Docker image for such purposes. To run it, please follow these instructions:
+
+  1. Install Docker
+
+  If not already installed on your machine, download and install Docker from [https://www.docker.com/](https://www.docker.com/).
+
+  2. Run Jekyll with Docker
+
+  Make sure to start docker before running the following command in the root directory of your local clone of this repository:
+
+  ```bash
+docker run --rm --volume="$PWD:/srv/jekyll:Z" --publish 4000:4000 jekyll/jekyll jekyll serve
```

Comment:
Could you add an explanation of the command here including the options (--rm, --volume, --publish), so people not genuinely familiar with docker commands could at least get an understanding what the command is doing (instead of blindly trusting it ;)

Reply:
Running unexplained shell commands found on the internet is what keeps things spicy!

Reply:
Tested with DOcker locally. Works also on Windows.

Please note that Windows Users using Git Bash for Windows will need to prefix this command with `MSYS_NO_PATHCONV=1` (see https://stackoverflow.com/questions/7250130/how-to-stop-mingw-and-msys-from-mangling-path-names-given-at-the-command-line).

Also, please add a note about where to find the website after running the docker command, like above with native jekyll: `Once it's ready, you can open your browser and visit `http://localhost:4000` to see your local version of the website.`